src/main/java/com/puppycrawl/tools/checkstyle/site/PropertiesMacro.java

```diff
@@ -158,13 +162,34 @@ private static void writeTablePropertiesRows(Sink sink)
         final Map<String, DetailNode> propertiesJavadocs = SiteUtil
                 .getPropertiesJavadocs(properties, currentModuleName, currentModuleFile);
 
-        for (String property : properties) {
+        final Set<String> validProperties = orderProperties(properties);
+
+        for (String property : validProperties) {
             final DetailNode propertyJavadoc = propertiesJavadocs.get(property);
             final DetailNode currentModuleJavadoc = propertiesJavadocs.get(currentModuleName);
             writePropertyRow(sink, property, propertyJavadoc, instance, currentModuleJavadoc);
         }
     }
 
+    /**
+     * Rearrange properties alphabetically with property 'tokens' always last.
+     *
+     * @param parameters the javadoc properties.
+     * @return LinkedHashSet contains valid properties for the javadoc.
+     *
+     */
+    private static Set<String> orderProperties(Set<String> parameters) {
+
+        final Set<String> validProperties = new LinkedHashSet<>(parameters);
+
+        if (validProperties.remove(TOKENS_PROPERTY)) {
+            validProperties.add(TOKENS_PROPERTY);
+        }
+
+        return validProperties;
```

Comment:
Please do modification to `properties` and return immutable set (or even better, a list since we have no need for set operations after this).

Reply:
@nrmancuso Changed in new PR